src/main/java/uk/gov/hmcts/reform/civil/controllers/claims/ClaimsController.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.civil.controllers.claims;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.hmcts.reform.civil.helpers.CaseDetailsConverter;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.CoreCaseDataService;
+
+@Api
+@Slf4j
+@RestController
+@AllArgsConstructor
+@RequestMapping(
+    path = "/claims",
```

Comment:
We should probably confirm the paths with HMCTS. Other possible options:
- /cases
- /claim-cases

Reply:
who could help us ?

Reply:
We can discuss this with HMCTS during the demo

Reply:
we will use /cases